src/include/pbs_config.h.in

```diff
@@ -632,6 +632,9 @@
 /* Defined when swig is available */
 #undef SWIG
 
+/* Python executable path */
+#undef SYSTEM_PYTHON_PATH
+
```

Comment:
Why this change in this bug fix?

Reply:
Is this is generated file or part of the src tree?

Reply:
yes, It is one of the GNU autotool generated files.  Its not related to this bug fix. I ran autogen and included the changes to retain consistency with branch.

Reply:
Sounds good